src/test/java/com/spring/springdata/SpringDataApplicationTests.java

```diff
@@ -0,0 +1,90 @@
+package com.spring.springdata;
+
+import com.spring.springdata.config.ApplicationConfig;
+import com.spring.springdata.entity.Course;
+import com.spring.springdata.entity.Student;
+import com.spring.springdata.servise.StudentService;
+import lombok.NoArgsConstructor;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.time.LocalDate;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = ApplicationConfig.class)
```

Comment:
for future: dont use your prod database for testing
use tests db

Reply:
Ok, thank you)